.travis.yml

```diff
@@ -3,12 +3,17 @@ language: go
 go:
   - "1.10"
 
+dist: jammy
```

Comment:
for the other examples, you declare the `language: java,` why differentiate here?

Reply:
because the `go` language is already declared as it's a go project (`language: go`)